extensions/amp-fit-text/0.1/test/test-amp-fit-text.js

```diff
@@ -53,13 +53,14 @@ describes.realWin(
         .then(() => ft);
     }
 
-    it('renders', () => {
+    it('renders', async () => {
       const text = 'Lorem ipsum';
-      return getFitText(text).then((ft) => {
+      return getFitText(text).then(async (ft) => {
```

Comment:
If this is moving to an async method, you can use `await` syntax instead of relying on the thenable directly.